_layouts/significant-change-pilot.html

```diff
@@ -42,6 +42,8 @@
       <div class="full-col tablet:grid-col-9 tablet:grid-offset-3 nav-main-section">
 		<div =id="updates" class="">
 			<h2>Updates</h2>
+			<h3 id="jan-2025">January 16, 2025</h3>
+			<p>FedRAMP is pleased to announce that the agile delivery pilot phase 1 has successfully concluded as of December 31st. The next step is the publication of a white paper that makes recommendations for policy updates to the SCR process, informed by data generated by the pilot. The target publication date for the agile delivery white paper is January 31st, so stay tuned!</p>
```

Comment:
Minor, do you want to capitalize ADP?

```suggestion
			<p>FedRAMP is pleased to announce that the Agile Delivery Pilot Phase 1 has successfully concluded as of December 31st. The next step is the publication of a white paper that makes recommendations for policy updates to the SCR process, informed by data generated by the pilot. The target publication date for the agile delivery white paper is January 31st, so stay tuned!</p>
```